src/layer/AbstractLayer.ts

```diff
@@ -79,4 +79,13 @@ export class AbstractLayer {
     }
     return flyoverIntervals;
   }
+
+  public async findDates(
+    bbox: BBox, // eslint-disable-line @typescript-eslint/no-unused-vars
+    fromTime: Date, // eslint-disable-line @typescript-eslint/no-unused-vars
+    toTime: Date, // eslint-disable-line @typescript-eslint/no-unused-vars
+    // any additional
+  ): Promise<Date[]> {
+    throw new Error('Not implemented yet');
```

Comment:
I noticed that EOCloud is not supported in this PR yet, do you think we should add it in a separate one?

I don't think the interface will change when we add it, so I don't see a problem, just asking. Fftr.

Reply:
Yes, still needs to be added. It can be in this PR I think.

Reply:
Support for EOCloud layers was added